ui/js/compareperf.js

```diff
@@ -5,13 +5,46 @@
 "use strict";
 
 perf.controller('CompareChooserCtrl', [
-  '$state', '$stateParams', '$scope', 'ThRepositoryModel', 'ThResultSetModel',
-  function CompareChooserCtrl($state, $stateParams, $scope,
+  '$state', '$stateParams', '$scope', '$http', 'ThRepositoryModel', 'ThResultSetModel',
+  function CompareChooserCtrl($state, $stateParams, $scope, $http,
                               ThRepositoryModel, ThResultSetModel) {
     ThRepositoryModel.get_list().success(function(projects) {
       $scope.projects = projects;
       $scope.originalProject = $scope.newProject = projects[0];
+      $scope.oriTipList = [];
```

Comment:
ori -> original, everywhere in the file. It's confusing to use abbreviations in one place and not others.